index.js

```diff
@@ -1,71 +1,106 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1
+counter = 2
+
+console.log("%cQuestion 1", "color: red") 
+
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
+const name = 'Radiyah'
+
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drinkWater(num) {
+   console.log("Man I sure am thirsty");
+   newNum = num -= 1 ;
+   console.log("Ahh that hits the spot");
+   console.log(newNum)
+}
```

Comment:
Although this works, it's not good practice to assign variables like this. If you want to create a new variable, I would suggest:
```js
  let newNum = num -1;
```

In this case, you don't necessarily need a new variable though because you already have the variable `num` that you can re-assign.
```suggestion
   num -= 1 ;
   console.log("Ahh that hits the spot");
   return num
```